voxygen/tests/validate_vox_files.rs

```diff
@@ -0,0 +1,13 @@
+extern crate dot_vox;
+use std::fs;
+
+#[test]
+fn validate_vox_files() {
+    let paths = fs::read_dir("./vox").unwrap();
+    
+    for path in paths {
+        let path_string = path.unwrap().path().into_os_string().into_string().unwrap();
+        let vox = dot_vox::load(&path_string);
+        assert_eq!(true, vox.is_ok(), "Failed to validate file '{:?}'", path_string);
+    }
```

Comment:
can you put a assert here that at least 1 file is checked in case the folder changes or so?

Reply:
on the other hand this should already be checked by the unwrap in line 6 or?

Reply:
Yes, it is already checked there

Reply:
Would it make sense to test the folder not being empty?

Reply:
yeah might be, can you adjust the test ?